app/helpers/application_helper/toolbar/chargeback_rate_center.rb

```diff
@@ -11,7 +11,7 @@ class ApplicationHelper::Toolbar::ChargebackRateCenter < ApplicationHelper::Tool
           'pficon pficon-edit fa-lg',
           t = N_('Edit this Chargeback Rate'),
           t,
-          :url_parms    => "main_div",
+          :url          => '/edit',
           :send_checked => true,
```

Comment:
We should probably remove send_checked, it looks like `url` starting with a `/` when on a screen with `recordId` will generate the right `/edit/123` url, so no need to add `miq_grid_checks`.